### PR TITLE
Deploy artifacts on Maven Central

### PR DESCRIPTION
**Pull request by TurpIF**
JIRA: DPP-3180

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -56,24 +63,18 @@ addPublication("Apk") {
     }
 
     android.applicationVariants.all {
+        val isMainVariant = name == mainVariant && false // TODO
```
Comment (leonardlabat) [inline app/build.gradle.kts:66]: TODO ?

File: publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java [hunk 2]
```diff
@@ -29,7 +29,7 @@
    * The hashing should be the users’ email address:
    * <ul>
    *   <li>Encoded in UTF-8</li>
-   *   <li>Trimmed of any white space (eg: “test@criteo.com “ should become “test@criteo.com”)</li>
+   *   <li>Trimmed of any white space (eg: “test@criteo.com “ should become “test@criteo.com“)</li>
```
Comment (leonardlabat) [inline publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java:32]: Do you want this to be part of that pull request ?

File: app/build.gradle.kts (hunk 1 above)
Comment (TurpIF) [reply app/build.gradle.kts:66]: Ah, good catch. I was checking if one artifact without classifier was required or not. Answer: all artifacts in the test app have a classifier: debug, staging, release, source, javadoc

File: publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java (hunk 2 above)
Comment (TurpIF) [reply publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java:32]: When pushing to maven local, the javadoc task was complaining about illegal character. I don't know why it is not working locally. But doing a PR dedicated for is kind of overkill. Let me put this in a dedicated commit if it's ok for you.

Comment (leonardlabat) [reply publisher-sdk/src/main/java/com/criteo/publisher/context/UserData.java:32]: Thanks for the explaination !

**Review by leonardlabat — APPROVED** [review]

**Pull request merged**